openid-4-verifiable-presentations-1_0.md

```diff
@@ -671,6 +671,12 @@ The error response follows the rules as defined in [@!RFC6749], with the followi
 
 Usage of `client_metadata` or `client_metadata_uri` parameters with `client_id` that the Wallet might be seeing for the first time is mutually exclusive with the registration mechanism where Self-Issued OP assigns `client_id` to the Verifier after receiving Verifier metadata.
 
+`access_denied`:
+
+- The Wallet did not have the requested Credentials to satisfy the Authorization Request.
```

Comment:
is this really `access_denied`..?

Reply:
What other error would you suggest? I would not create a dedicated error for privacy reasons.

Reply:
yeah, i don't really know an alternative, `access_denied` feels good enough after the WG discussion.